src/algokit_utils/beta/composer.py

```diff
@@ -0,0 +1,657 @@
+
+        if params.static_fee is not None and params.extra_fee is not None:
+            raise ValueError('Cannot set both static_fee and extra_fee')
+
+        if params.static_fee is not None:
+            txn.fee = params.static_fee
+        else:
+            txn.fee = txn.estimate_size() * suggested_params.fee or algosdk.constants.min_txn_fee
+            if params.extra_fee:
+                txn.fee += params.extra_fee
+
+        if params.max_fee is not None and txn.fee > params.max_fee:
+            raise ValueError(f'Transaction fee {txn.fee} is greater than max_fee {params.max_fee}')
+
+        return txn
+    
+    def _build_payment(self, params: PayParams, suggested_params: algosdk.transaction.SuggestedParams) -> algosdk.transaction.Transaction:
+        txn = algosdk.transaction.PaymentTxn(
+            sender=params.sender,
+            sp=suggested_params,
+            receiver=params.receiver,
+            amt=params.amount,
+            close_remainder_to=params.close_remainder_to,
+        )
+
+        return self._common_txn_build_step(params, txn, suggested_params)
+
+    def _build_asset_create(self, params: AssetCreateParams, suggested_params: algosdk.transaction.SuggestedParams) -> algosdk.transaction.Transaction:
+        txn = algosdk.transaction.AssetConfigTxn(
+            sender=params.sender,
+            sp=suggested_params,
+            total=params.total,
+            default_frozen=params.default_frozen or False,
+            unit_name=params.unit_name,
+            asset_name=params.asset_name,
+            manager=params.manager,
+            reserve=params.reserve,
+            freeze=params.freeze,
+            clawback=params.clawback,
+            url=params.url,
+            metadata_hash=params.metadata_hash,
+            decimals=params.decimals or 0,
+            strict_empty_address_check=False,
+        )
+
+        return self._common_txn_build_step(params, txn, suggested_params)
+
+    def _build_app_call(self, params: AppCallParams, suggested_params: algosdk.transaction.SuggestedParams) -> algosdk.transaction.Transaction:
+        sdk_params = {
+            'sender': params.sender,
+            'sp': suggested_params,
+            'index': params.app_id or 0,
+            'on_complete': params.on_complete or algosdk.transaction.OnComplete.NoOpOC,
+            'approval_program': params.approval_program,
+            'clear_program': params.clear_program,
+            'app_args': params.args,
+            'accounts': params.account_references,
+            'foreign_apps': params.app_references,
+            'foreign_assets': params.asset_references,
+            'extra_pages': params.extra_pages,
+            'local_schema': algosdk.transaction.StateSchema(num_uints=params.schema.get("local_uints") or 0, num_byte_slices=params.schema.get("local_byte_slices") or 0) if params.schema else None,
```

Comment:
```suggestion
            'local_schema': algosdk.transaction.StateSchema(num_uints=params.schema.get("local_uints", 0), num_byte_slices=params.schema.get("local_byte_slices", 0)) if params.schema else None,
```

and similarly for global_schema